docs/contributing.md

```diff
@@ -43,14 +41,14 @@ Pydantic has few dependencies, doesn't require compiling and tests don't need ac
 Because of this, setting up and running the tests should be very simple.
 
 !!! tip
-    **tl;dr**: use `make format` to fix formatting, `make` to run tests and linting & `make docs`
+    **tl;dr**: use `make format` to fix formatting, `make` to run tests and linting and `make docs`
     to build the docs.
 
 ### Prerequisites
 
 You'll need the following prerequisites:
 
-- Any Python version between **Python 3.8 and 3.11**
+- Any Python version between **Python 3.9 and 3.12**
```

Comment:
Figured we should recommend at least 3.9 as we might have some dev dependencies requiring 3.9 (was able to remove the one we were talking to yesterday, but might happen in the future).